packages/contentstack-import/src/import/modules-js/environments.js

```diff
@@ -82,11 +82,7 @@ module.exports = class ImportEnvironments {
               });
           } else {
             // the environment has already been created
-            log(
-              config,
-              `The environment ${env.name} already exists. Skipping it to avoid duplicates!`,
-              'success',
-            );
+            log(config, `The environment ${env.name} already exists. Skipping it to avoid duplicates!`, 'success');
```

Comment:
@cs-raj Please use the quotes for the entity name for better readability.

Reply:
Done